README.md

```diff
@@ -1 +1,111 @@
+
+### Nested Hashes Model Real-World Data
+
+We can imagine so many real-world situations and environments in which we are dealing with complicated collections of data. Take the example of a simple address book. 
+
+An individual contact from address book can be stored in a simple hash:
+
+```ruby
+zoe = {name: "Zoe", phone: "917-555-9875", address: "24 W. 124th Street, NY, NY"}
+``` 
+
+However, an address book contains more than one contact. We could collect all of our individual person hashes and store them in a `contacts` array:
+
+```ruby
+contacts = [
+	{last_name: "DeBenedetto", first_name: "Zoe", phone: "917-555-9875"}, 
+	{last_name: "DeBenedetto", first_name: "Richard", phone: "917-555-0912"}
+	{last_name: "Londin", first_name: "Victoria", phone: "917-555-0912"}
+]
+```
+
+This works, but it's not a great model of a real address book. In a real address book, contacts are sorted, or grouped, usually by last name. In our collection of hashes above, there is no way to organize our data like that. 
+
+This is where nested hashes come in. 
+
+## Our First Nested Hash
+
+With a nested hash, we can create a key that points to a value of *another hash or an array*. Let's take a look:
+
+```ruby
+contacts = {
+	"D" =>[
+			{last_name: "DeBenedetto", first_name: "Zoe", phone: "917-555-9875"}, 
+			{last_name: "DeBenedetto", first_name: "Richard", phone: "917-555-0912"}
+		  ], 
+	"L" => [
+			{last_name: "Londin", first_name: "Victoria", phone: "917-555-0912"}
+		  ]
+}
+```
+
+Nested hashes allow us to further group, or associate, the data we are working with. They help us to deal with situations in which a category or piece of data is associated not just to one discrete value, but to a collection of values. In such a situation, we can create a hash key that points to a value of another hash or an array. 
+
+### A Note On Mixing our Collection Types
+
+You may have noticed in the above examples that we used both an array of hashes and a hash in which the value of a key is an array. 
+
+Understand that arrays and hashes can store *any type of data*. In other words, the individual index items of an array can be strings, integers, even other arrays and hashes. The same is true of hashes. The values that hash keys point to may be strings, integers, even arrays and hashes. 
+
+
+## Nested Hash Example
+
+We'll be building up our own nested hash and operating on such data structures in a number of ways over the course of the next few lessons. For now, just read through the next example and get comfortable looking at a nested hash. 
+
+In this example, we have a hash, `flatiron_school`, which stores some data about us. This data is broken down into the categories of `:instructors`, `:dev_team` and `:students`, thanks to our nested hash. 
```

Comment:
You now have three different "domains" you've talked about. Maybe bring this teacher domain up into the previous example?